tools-v2/pkg/cli/command/curvebs/list/client/client.go

```diff
@@ -146,11 +147,22 @@ func (pCmd *ClientCommand) RunCommand(cmd *cobra.Command, args []string) error {
 	pCmd.TableNew.AppendBulk(list)
 	errRet := cmderror.MergeCmdError(errors)
 	pCmd.Error = errRet
-	pCmd.Result = results
+	pCmd.Result = rows
 	return nil
 }
 
 // ResultPlainOutput implements basecmd.FinalCurveCmdFunc
 func (pCmd *ClientCommand) ResultPlainOutput() error {
 	return output.FinalCmdOutputPlain(&pCmd.FinalCurveCmd)
 }
+
+func GetClientList(caller *cobra.Command) (*interface{}, *tablewriter.Table, *cmderror.CmdError) {
```

Comment:
```go
type ClientCommand struct {
	basecmd.FinalCurveCmd
	Rpc             []*ListClientRpc
    clientinfoList  []*nameserver2.ClientInfo
}
```

```suggestion
func GetClientList(caller *cobra.Command) ([]*nameserver2.ClientInfo, *cmderror.CmdError) {
```

Reply:
so i also need to parse the `GetclientList` rpc response in `curve bs status client`?

Reply:
row or response is better than table

Reply:
done